test/integration/api/AccountingAPI.spec.ts

```diff
@@ -217,22 +217,16 @@ describe('AccountingAPI', () => {
     });
 
     describe('when the caller does not have enough balance of token', async () => {
-      let userTokenBalance: BigNumber;
-      let token: StandardTokenMockContract;
-
       beforeEach(async () => {
-        token = _.first(tokens);
-        subjectTokenAddressesToDeposit = [token.address];
+        const depositToken = _.first(tokens);
+        subjectTokenAddressesToDeposit = [depositToken.address];
         subjectQuantitiesToDeposit = [depositQuantity];
         subjectCaller = ACCOUNTS[1].address;
       });
 
       test('throws', async () => {
-        userTokenBalance = await token.balanceOf.callAsync(subjectCaller);
-        return expect(subject()).to.be.rejectedWith(
-    `User has balance of ${userTokenBalance} when required balance is ${depositQuantity}. Increase user's
-    token balance at token address ${token.address}`
-        );
+        // TODO - Can add rejection message after promise race conditions are fixed
```

Comment:
Did you say you had a solution to the race condition last week?

Reply:
My fix was to remove the specific check for now until we refactor our test setup